spec/stat_tracker_spec.rb

```diff
@@ -38,17 +36,47 @@
 
     it "finds highest total score" do
       expect(@stat_tracker.highest_total_score).to eq(7)
-   end
+    end
 
     it 'can return the lowest score' do
       expect(@stat_tracker.lowest_total_score).to eq(1)
     end
 
-    it "returns the percentage of tied games" do
-      expect(@stat_tracker.percentage_ties).to eq(17.8)
+    it "finds highest total score" do
+      expect(@stat_tracker.highest_total_score).to eq 7
```

Comment:
Let's be consistent with using `()`'s, i.e. `....to eq(7)`

Reply:
good catch! thanks